modules/ROOT/pages/fix-instance-conformance-issues.adoc

```diff
@@ -0,0 +1,63 @@
+= Finding and Fixing Conformance Issues in API Instances
+
+After API Governance identifies conformance issues in your API instances, you can view the governance validation report details to get the information you need to fix the issues. After you identify what needs to be fixed, go to the instance in API Manager and fix the issue. If the issue is an instance-related issue in the specification, use your API editor, such as API Designer, to fix the specification.
+
+// Screenshot of a Conformance Report page with callouts 
+
+include::_partials/api-gov-apim-governance-report.adoc[]
+
+== View Conformance Issue Details in the Governance Report
+
+To view instance conformance issues in the governance validation report:
```

Comment:
Just a thought... I wonder if a screenshot (or two) with callouts (instead of the hierarchy) would be easier to see at a glance.

Reply:
I'm not sure I understand this comment. I have two screenshots with callouts in the topic. They're in partials. You can see the built doc [here on the internal site](https://dev-docs-internal.kqa.msap.io/beta-apig-apim/api-governance/fix-instance-conformance-issues#example-rules-and-recommended-fixes).

Reply:
Do you mean side by side vs. in separate sections? BTW, we will be building up this functionality and corresponding doc in future releases, but it's MVP for this release.